EntertainmentCost.java

```diff
@@ -0,0 +1,33 @@
+import java.util.Scanner;
+
+public class EntertainmentCost {
+    public static void main(String[] args) {
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.print("Enter age: ");
+        int age = scanner.nextInt();
+        scanner.nextLine();
+
+        System.out.print("Is the customer VIP? (yes/no): ");
+        String vipStatus = scanner.nextLine().trim().toLowerCase();
+        boolean isVip = vipStatus.equals("yes");
+
+        final int ENTRANCE_FEE = 5000;
+        final int MIN_AGE = 16;
+
+        if (age < MIN_AGE) {
```

Comment:
So your code works only if the client's age is less than 16 :) read the requirements again.